src/applications/personalization/profile/components/direct-deposit/BankInfoCNP.jsx

```diff
@@ -250,9 +249,9 @@ export const BankInfoCNP = ({
       data.push({
         title: 'Payment history',
         value: (
-          <EbenefitsLink path="ebenefits/about/feature?feature=payment-history">
+          <a href="/va-payment-history/payments/" target="_blank">
```

Comment:
Do we actually want this to open a new tab/window? I think we tend to only do that when we are sending someone to a non-VA page.

Reply:
Great catch — it looks like the pattern is to open in the same tab/window for links within VA.gov, and to only open up a new tab or window if the link is outside of VA.gov. So, we should not open a new tab or window now that the link is going to another page within VA.gov.

Reply:
Thanks! I will make the update.